common/test/acceptance/pages/lms/bookmarks.py

```diff
@@ -3,6 +3,7 @@
 """
 from bok_choy.promise import EmptyPromise
 from .course_page import CoursePage
+from selenium.webdriver.common.keys import Keys
 
 
 class BookmarksPage(CoursePage):
```

Comment:
Please refactor to use the `PaginatedUIMixin` in `common/test/acceptance/pages/common/paging.py`